### PR TITLE
Add option for disabling Skeletron Prime bombs 

### PR DESCRIPTION
**Pull request by moisterrific**
this has the potential for large scale world destruction that bypasses region protection on `for the worthy` seed worlds

~~also not sure if I should add one for regular bombs since those can spawn randomly from trees and pots on that seed too~~

nvm I played around w/ trying to disable regular bombs and it doesn't rly work

?????? HAVE YOU UPDATED THE CHANGELOG? ??????
yes

### CODE REVIEW

File: TShockAPI/TShock.cs [hunk 1]
```diff
@@ -1618,15 +1618,19 @@ private void NpcHooks_OnStrikeNpc(NpcStrikeEventArgs e)
 		private void OnProjectileSetDefaults(SetDefaultsEventArgs<Projectile, int> e)
 		{
 			//tombstone fix.
-			if (e.Info == 43 || (e.Info >= 201 && e.Info <= 205) || (e.Info >= 527 && e.Info <= 531))
+			if (e.Info == ProjectileID.Tombstone || (e.Info >= ProjectileID.GraveMarker && e.Info <= ProjectileID.Obelisk) || (e.Info >= ProjectileID.RichGravestone1 && e.Info <= ProjectileID.RichGravestone5))
```
Comment (hakusaro) [inline TShockAPI/TShock.cs]: Can you move this change into a separate PR or commit that says what this change is?

File: TShockAPI/TShock.cs [hunk 2]
```diff
@@ -1618,15 +1618,19 @@ private void NpcHooks_OnStrikeNpc(NpcStrikeEventArgs e)
...
 				if (Config.Settings.DisableTombstones)
 					e.Object.SetDefaults(0);
-			if (e.Info == 75)
+			if (e.Info == ProjectileID.HappyBomb)
```
Comment (hakusaro) [inline TShockAPI/TShock.cs]: Can you move this change into a separate PR or commit that says what this change is?

File: TShockAPI/TShock.cs [hunk 3]
```diff
@@ -1618,15 +1618,19 @@ private void NpcHooks_OnStrikeNpc(NpcStrikeEventArgs e)
...
 				if (Config.Settings.DisableClownBombs)
 					e.Object.SetDefaults(0);
-			if (e.Info == 109)
+			if (e.Info == ProjectileID.SnowBallHostile)
```
Comment (hakusaro) [inline TShockAPI/TShock.cs]: Can you move this change into a separate PR or commit that says what this change is?

File: CHANGELOG.md [hunk 4]
```diff
@@ -32,6 +32,7 @@ This is the rolling changelog for TShock for Terraria. Use past tense when addin
 * Added `summonboss` permission check for Prismatic Lacewing. Players who do not have said permission will be unable to kill this critter, as it will summon the Empress of Light. Also added support for the `AnonymousBossInvasions` config option, if this is set to `false` it will now broadcast the name of the player who summoned her. (@moisterrific)
 * Added `ForceTime` config setting check for Enchanted Sundial usage. If `ForceTime` is set to anything other than `normal`, Sundial use will be rejected as this would lead to very janky game behavior. Additionally, players with `cfgreload` permission will be advised  to change it back to `normal` in order to use sundial. (@moisterrific, @bartico6)
 * Added `%onlineplayers%` and `%serverslots%` placeholders for MOTD. The default MOTD message was also updated to use this. (@moisterrific, @bartico6)
+* Added config option `DisablePrimeBombs` for disabling Skeletron Prime bombs from spawning, as these bombs would bypass region protection and destroy tiles on `for the worthy` seed worlds. (@moisterrific)
```
Comment (hakusaro) [inline CHANGELOG.md]: Can you move this to the new upcoming changes section? Sorry, technical difficulties resulted in a release before this PR was merged.

**Review by hakusaro — CHANGES_REQUESTED** [review]
Can you split or otherwise isolate changes unrelated to skeletron prime bombs into another commit or PR? If not, please note that someone needs to manually check this for accuracy.

Also can you move the changelog item to the new upcoming changes section? Other than that, this is good! Thank you so much for your pull request!

**Comment by moisterrific** [comment]
> 
> 
> Can you split or otherwise isolate changes unrelated to skeletron prime bombs into another commit or PR? If not, please note that someone needs to manually check this for accuracy.

yeah sorry about that, I saw the other ones were using item IDs instead of more human readable `Terraria.ID` names, so I thought I might as well update them while I'm adding this new thing lol

**Comment by moisterrific** [comment]
oops I think I did something wrong
